<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>410</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel#label_top{

}
QLabel#label_title{
	border:none;
	image: url(:/res/title.png);

}
.QWidget#widget{
	background-color: rgb(255,255,255);
}
QPushButton#btn_min{
	border:none;
	image: url(:/res/zuixiaohua.png);
}
QPushButton#btn_passwordview{
	border:none;
}
QPushButton#btn_close{
	border:none;
	image: url(:/res/guanbi.png);
}
QPushButton#btn_min:hover{
	background-color: rgba(190 ,190 ,190 ,100);
	
}
QPushButton#btn_close:hover{
	background-color: rgb(255, 0, 0)
}
QPushButton#btn_more{
	border:none;
	
	image: url(:/res/more.png);
}
QPushButton#btn_more:hover{
	background-color: rgba(190 ,190 ,190 ,100);
}
QPushButton#btn_login{
	border:none;
	background-color: rgb(11,194,255);
	border-radius:5px;
	font:normal 21px &quot;楷体&quot;;
	color:white
}
QPushButton#btn_qrcode{
	border:none;
	image: url(:/res/erweima.png);
}
QPushButton#btn_resiger{
	border:none;
}
QPushButton#btn_find_password{
	border:none;
}
QComboBox#comboBox_account,QLineEdit#password,QLineEdit#account{
	border:none;
	border-bottom:1px solid rgb(229,229,229);
	font:normal 17px &quot;楷体&quot;;
	color:rgb(154,154,154)
}
QComboBox#comboBox_account:hover,QLineEdit#password:hover,QLineEdit#account:hover{
	border-bottom:1px solid rgb(193,193,193);
}
QLineEdit#password:focus,QLineEdit#account:focus{
	border-bottom:1px solid rgb(18,183,245);
}
QComboBox#comboBox_account::drop-down{
	min-width:30px;
	border-image: url(:/res/jietou.png);
}
QComboBox#comboBox_account::drop-down:checked{
	border-image: url(:/res/jietou_hover.png);
}
QComboBox#comboBox_account QAbstractItemView{
	border:none;
	background-color:white;
	outline:0px;
}
</string>
  </property>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>510</width>
     <height>410</height>
    </rect>
   </property>
   <widget class="QLabel" name="label_head">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>100</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/res/head.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>160</y>
      <width>510</width>
      <height>241</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>510</width>
      <height>235</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1000</width>
      <height>1000</height>
     </size>
    </property>
    <widget class="QPushButton" name="btn_resiger">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>200</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>注册账号</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_qrcode">
     <property name="geometry">
      <rect>
       <x>451</x>
       <y>191</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_login">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>150</y>
       <width>271</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>登录</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_account">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>135</x>
       <y>32</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="password">
     <property name="geometry">
      <rect>
       <x>135</x>
       <y>74</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>密码</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_username">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>40</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="res.qrc">:/res/username.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_password">
     <property name="geometry">
      <rect>
       <x>98</x>
       <y>75</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="res.qrc">:/res/password.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>110</y>
       <width>271</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QCheckBox" name="checkBox_login">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>自动登录</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_savepassword">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>记住密码</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_find_password">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>找回密码</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLineEdit" name="account">
     <property name="geometry">
      <rect>
       <x>135</x>
       <y>32</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>QQ账号/手机/邮箱</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_passwordview">
     <property name="geometry">
      <rect>
       <x>365</x>
       <y>80</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/res/password1.png</normaloff>:/res/password1.png</iconset>
     </property>
    </widget>
    <zorder>password</zorder>
    <zorder>comboBox_account</zorder>
    <zorder>btn_resiger</zorder>
    <zorder>btn_qrcode</zorder>
    <zorder>btn_login</zorder>
    <zorder>label_username</zorder>
    <zorder>layoutWidget</zorder>
    <zorder>btn_passwordview</zorder>
    <zorder>account</zorder>
    <zorder>label_password</zorder>
   </widget>
   <widget class="QLabel" name="label_top">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>7</y>
      <width>510</width>
      <height>153</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>420</width>
      <height>132</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1000</width>
      <height>1000</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/res/mian.gif</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_title">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_more">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>8</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1000</width>
      <height>1000</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_close">
    <property name="geometry">
     <rect>
      <x>469</x>
      <y>10</y>
      <width>35</width>
      <height>35</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1000</width>
      <height>1000</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_min">
    <property name="geometry">
     <rect>
      <x>425</x>
      <y>10</y>
      <width>35</width>
      <height>35</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1000</width>
      <height>1000</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label_top</zorder>
   <zorder>widget</zorder>
   <zorder>label_title</zorder>
   <zorder>label_head</zorder>
   <zorder>btn_more</zorder>
   <zorder>btn_close</zorder>
   <zorder>btn_min</zorder>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
